packages/api/src/models/medical/ihe-result.ts

```diff
@@ -15,7 +15,7 @@ export abstract class BaseIHEResultModel<T extends Model<any, any>> extends Mode
 > {
   declare id: string;
   declare requestId: string;
-  declare patientId: string;
+  declare patientId: string | undefined;
```

Comment:
Why is this getting or undefined? We would need to change the schema and the migration

Reply:
I believe its actually optional which is why I updated it. Maybe @ShamilN can pitch in here?

Reply:
@leite08 sure. Here its optional: https://www.notion.so/metriport/XCA-ITI-38-Interface-Outgoing-DQ-0130d18144da45969bb80f179ea43335
In the iti39 its marked as required. Note that this patientID is not the same as the gatewayPatientId.id which contains our generated id which we actually use for search. This patientId is from the FHIR patient resource I believe.

Reply:
The _patientId_ in 38 is for logging and reference purposes only, to trace the actual patient ID. 
It is not used anywhere in the actual request.

Reply:
I had to create a [thread](https://metriport.slack.com/archives/C065BLRBUDQ/p1703810859879409) to try to understand this. There's only one `patientId` and this is Metriport's. And that's the same as the FHIR Server's Patient ID. So this is all very confusing.

Reply:
renaming gatewayPatientId.id to externalGatewayPatient.id

Reply:
and resolved things so we dont need a DB migration